Podfile

```diff
@@ -170,9 +170,9 @@ target 'WordPress' do
     ##
 
     # Production
-    pod 'Automattic-Tracks-iOS', '~> 0.4.3'
+    # pod 'Automattic-Tracks-iOS', '~> 0.4.3'
```

Comment:
Please change this PR to reference a version instead of a tag prior to merging :)